### ISSUE TITLE
Error reading PDB files

### ISSUE THREAD

**Issue by niranjchandrasekaran** [niranjchandrasekaran/PyPath#7]
While read PDB files, Charlie got the following error
``` 
pdb = [data[line] for line in range(len(data)) if (data[line][2].rstrip()).lstrip() == 'CA']
IndexError: list index out of range
```

**Comment by niranjchandrasekaran** [comment]
I was able to reproduce the error and I believe it is probably due to missing/additional columns in the PDB file

**Comment by niranjchandrasekaran** [comment]
The error came from the code reading lines other than that of `HETATM` and `ATOM`. This has now been fixed by letting the code read only these lines.

**Comment by niranjchandrasekaran** [comment]
While fixing this error, I also found that since PDB columns were being separated using the split() function, when there were no gaps between columns, (for e.g. between Chain Name and the Amino Acid number) the columns weren't being separated. This also ended up causing an error which was fixed by hard coding the column sizes defined in the [PDB format](https://www.cgl.ucsf.edu/chimera/docs/UsersGuide/tutorials/pdbintro.html).
